doc_source/serverless-policy-template-list.md

```diff
@@ -2155,6 +2156,43 @@ If you are not using default key, you will also need the `KMSDecryptPolicy` poli
         ]
 ```
 
+## SSMParameterWithSlashPrefixReadPolicy<a name="ssm-parameter-with-slash-prefix-read-policy"></a>
+
+Gives permission to access a parameters from an Amazon EC2 Systems Manager \(SSM\) parameter store to load secrets in this account\. Use when parameter name has slash prefix\.
```

Comment:
Is this sentence grammatically correct? Better to say:
> Gives permission to access parameters from an ...

Reply:
Good catch. It gives access to a singular parameter so it should be `Gives permission to access a parameter from an ...`.